src/server/game/Entities/Player/Player.cpp

```diff
@@ -10080,6 +10080,20 @@ uint32 Player::GetItemCountWithLimitCategory(uint32 limitCategory, Item* skipIte
     return count;
 }
 
+std::vector<Item*> Player::GetCraftingReagentItems()
+{
+    std::vector<Item*> itemList = std::vector<Item*>();
+    ForEachItem(ItemSearchLocation::Equipment | ItemSearchLocation::Inventory, [&itemList](Item* item)
```

Comment:
can we equip crafting reagents?

Reply:
you are right idk why i was thinking about enchantment rods equipped